jax_unirep/evotuning_models.py

```diff
@@ -23,12 +23,18 @@
 """
 from jax.experimental.stax import Dense, Softmax, serial
 
-from .layers import mLSTM, mLSTMHiddenStates
+from .layers import AAEmbedding, mLSTM, mLSTMHiddenStates
 
 
 def mlstm1900():
     """mLSTM1900 model functions."""
```

Comment:
Since we're introducing the ability to create any arbitrarily-sized mLSTM with arbitrary amino acid embedding sizes, we probably can distinguish this function to be the "mLSTM1900 architecture from original UniRep paper".

Here's a proposed new docstring:

```suggestion
    """Return mLSTM1900 model's initialization and forward pass functions.
    
    The initializer function returned will give us random weights as a starting point.
    
    The model forward pass function will accept any weights compatible with those
    generated by the initializer function.
    The model implemented here has a trainable embedding,
    one mLSTM layer with 1900 nodes,
    and a single dense layer to predict the next amino acid identity.
    
    This model is also the default used in `get_reps`.
    """
```